src/app/components/BibPage/LibraryHoldings.jsx

```diff
@@ -7,6 +7,16 @@ const LibraryHoldings = ({ holdings }) => {
     return null;
   }
 
+  const liForEl = (el) => {
+    if (typeof el === 'string') return el;
+    if (!el.url) return el.label;
+    return (
+      <a href={el.url}>
+        { el.label }
+      </a>
```

Comment:
Doesn't this still need to be wrapped in an `<li>`?

Reply:
Fixed